### ISSUE TITLE
Secret Loss is not being optimized

### ISSUE THREAD

**Issue by haroonkhalidmughal** [tancik/StegaStamp#53]
Hi Tancik, 

I have utilized your repository for training multiple models with various hyper-parameter changes, however, one thing that is common in all of my trainings is that the secret loss (cross entropy) isnt being optimized in all of them or there is a minute optimization resulting in a really low str_acc. 

Do you know what could be the possible issue for that?

**Comment by BeneyKim** [comment]
I have same problem. 
@haroonkhalidmughal  Did you solve this problem?

**Comment by liudan193** [comment]
I also have this problem. Do you solve it? @BeneyKim @haroonkhalidmughal
